core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -547,6 +547,10 @@ <h2 class="activity-title e2e-test-exp-summary-tile-title">
     position: relative;
   }
 
+  .oppia-dashboard-card-view-item:hover{
```

Comment:
Hey @hemantmm, if you just take a look at the failed tests and click on details, it will take to the logs of the test. Once you scroll all the way to the bottom, you can see the error.

![image](https://user-images.githubusercontent.com/70763993/224151872-8c8a97f8-8015-44cd-bfba-ee8c4a942d5a.png)
 The error is `550:39  ✖  Expected single space before "{"  block-opening-brace-space-before`. It means that here you need to add a space between `hover` and `{`

You can also run lint checks locally, for this you can refer to this https://github.com/oppia/oppia/wiki/Lint-Checks.